src/SDKs/DBforPostgreSQL/AzSdk.RP.props

```diff
@@ -0,0 +1,7 @@
+﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
@rohit-joy rename the root directory for your SDK to PostgreSql instead of DBforPostgreSql

Reply:
Done